recipes-zarhus/images/zarhus-base-image.inc

```diff
@@ -1,4 +1,4 @@
-DESCRIPTION = "zarhus image"
+SUMMARY = "zarhus image"
```

Comment:
If we care on the spelling, this could also be checked automatically and placed under https://github.com/3mdeb/hooks

For instance, coreboot is very sensitive on this: https://doc.coreboot.org/#spelling-of-coreboot

ChatGPT suggests something like (not tested)

```
import sys
import re

TRADEMARK_RULES = {
    'Zarhus': 'Zarhus',
    'Dasharo': 'Dasharo',
    'coreboot': 'coreboot',
    'Yocto': 'Yocto',
}

def check_and_fix_trademark(filename, autofix=False):
    with open(filename, 'r') as file:
        content = file.read()

    fixed_content = content
    found_issues = False

    for trademark, correct_format in TRADEMARK_RULES.items():
        pattern = re.compile(re.escape(trademark), re.IGNORECASE)
        matches = pattern.findall(content)
        for match in matches:
            if match != correct_format:
                found_issues = True
                print(f"{filename}: '{match}' should be '{correct_format}'")
                if autofix:
                    fixed_content = fixed_content.replace(match, correct_format)

    if found_issues and autofix:
        with open(filename, 'w') as file:
            file.write(fixed_content)

    return not found_issues

def main():
    autofix = '--autofix' in sys.argv
    files = [arg for arg in sys.argv[1:] if arg != '--autofix']
    all_passed = True
    for filename in files:
        if not check_and_fix_trademark(filename, autofix):
            all_passed = False
    if not all_passed:
        sys.exit(1)

if __name__ == "__main__":
    main()
```

Reply:
@macpijan Sounds good. Do you know if it's possible to create the hook without creating a new repo?